### PR TITLE
Add satip devices option

### PR DESCRIPTION
**Pull request by frankr612**
add option/setting to start satip vdr plugin with specified number of devices

### CODE REVIEW

**Comment by lrusak** [comment]
LGTM

@cvh what do you think?

**Comment by stefansaraev** [comment]
looks good. should be squashed down to one commit.

File: packages/addons/service/vdr-addon/source/resources/settings.xml [hunk 1]
```diff
@@ -32,6 +32,7 @@
 		<setting id="ENABLE_XMLTV2VDR" type="bool" label="1045" default="false" />
 		<setting id="ENABLE_DUMMYDEVICE" type="bool" label="1047" default="false" />
 		<setting id="ENABLE_SATIP" type="bool" label="1048" default="false" />
+		<setting id="SATIP_NUM_DEVICES" type="slider" option="int" range="1,4" label="1058" visible="!eq(-1,false)" default="1" />
```
Comment (CvH) [inline packages/addons/service/vdr-addon/source/resources/settings.xml]: @frankr612 could you adjust the range to 8 or 10 as at least 8 tuner Sat>IP devices are out there and self made are easily >4. If you are unaware how to squash your commits we can do this at merge time !

**Comment by frankr612** [comment]
You're right, my device has actually 8 tuners. Some may have even more, so I changed the range to 10 now and squashed the commits into 1.

**Comment by chewitt** [comment]
@frankr612 @CvH in future please prefix the commit message with "vdr:" so we know what it's related to when retro-browsing lists of commits, thanks.

**Comment by frankr612** [comment]
@chewitt ok, I will, thanks!

**Pull request merged**